recipe/meta.yaml

```diff
@@ -40,22 +40,138 @@ source:
 
 build:
   number: {{ build }}
+  string: cuda{{ cuda_compiler_version | replace('.', '') }}_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version != "None"]
+  string: cpu_{{ blas_impl }}_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}                                 # [cuda_compiler_version == "None"]
+  detect_binary_files_with_prefix: false
+  run_exports:
+    - {{ pin_subpackage('libtorch', max_pin='x.x') }}
+  ignore_run_exports_from:
+    - python *                               # [not osx]
```

Comment:
maybe it would be better to define a new variable like: megabuild

and condition on that? that way we could keep a comment as to why osx is not a megabuild.